src/components/NumberTicker.tsx

```diff
@@ -2,36 +2,33 @@ import * as React from 'react'
 import { Animated, StyleProp, StyleSheet, Text, TextStyle, View } from 'react-native'
 import { typeScale } from 'src/styles/fonts'
 
-interface CommonProps {
-  textHeight: number
-  textStyles?: StyleProp<TextStyle>
-  animationDuration?: number
-}
-interface Props extends CommonProps {
+interface Props {
   finalValue: string
```

Comment:
do we need to emphasize that this is indeed a _final_ value here?

Reply:
ah, good question. i guess not, but i started with this because i wanted to enable a `startValue`. for example, if we want to use this animation, maybe we want to only animate the changes from the first balance to the updated balance. i then decided against allowing a custom startValues because 1) after playing around with the animation, i think starting from a random number is not visually jarring and draws more emphasis to the changing data, 2) the logic for ensuring that the start value is compatible with the final value got very complex (trying to ensure the same number of digits, that the decimal is in the right place in both number, etc). so i can change this back to a simple `value`